### PR TITLE
chore: adding visual improvements and two components to adoption guide

### PR DESCRIPTION
**Pull request by bbogdanov**
Signed-off-by: Bogdan Bogdanov <bbogdanov@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [x] Other... Please describe:

Adoption guide

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/angular/projects/clarity-adoption/src/app/app.component.html [hunk 1]
```diff
@@ -3,10 +3,9 @@
   <header class="header-6">
     <div class="branding">
       <img
-        cds-layout="p:md"
         src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzYiIGhlaWdodD0iMzYiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PGcgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj48cGF0aCBmaWxsPSIjMTc5YmQzIiBkPSJNMjQuNzAyIDQuNTgzbDExLjI3OCA2LjY3OS0uMDE5IDEzLjU2My0xMS4yNiA2LjY2LTYuNjg0LTMuOTcgMTEuNjUyLTYuNTQxdi01Ljg5OGwtNC43NDctMi42MDQtNi44OS0zLjkzOCIvPjxwYXRoIGZpbGw9IiNGMzhCMDAiIGQ9Ik0xMS4zMzEgNC41ODNMLjA1NCAxMS4yNjJsLjAxOCAxMy41NjMgMTEuMjYgNi42NiA2LjY4NS0zLjk3LTEwLjY2My02LjU0MXYtNS44OThsMTAuNjc4LTYuNTQyIi8+PHBhdGggZmlsbD0iIzAwNjQ4ZiIgZD0iTTE4LjAxNyAyNy41MTVMMTEuNSAyMy41MTZsNi41MjItMy44NDUgNi45MyAzLjk1MiIvPjxwYXRoIGZpbGw9IiM5ODQ0MUUiIGQ9Ik0xOC4wMzEgOC41MzNsLTYuNTA0IDMuOTg1IDYuNTE3IDMuODg0IDYuODYyLTMuOTQiLz48L2c+PC9zdmc+"
       />
-      <a routerLink="/" class="nav-link"><span class="title">Clarity Adoption</span></a>
+      <a cds-layout="m-x:md" routerLink="/" class="nav-link"><span class="title">Clarity Adoption</span></a>
```
Comment (mathisscott) [inline packages/angular/projects/clarity-adoption/src/app/app.component.html:8]: 🏆

File: packages/angular/projects/clarity-adoption/src/app/app.component.html [hunk 2]
```diff
@@ -29,17 +28,21 @@
...
           <a routerLink="/button" routerLinkActive="active" clrVerticalNavLink>Button</a>
+          <a routerLink="/checkbox" routerLinkActive="active" clrVerticalNavLink>Checkbox</a>
           <a routerLink="/icons" routerLinkActive="active" clrVerticalNavLink>Icons</a>
+          <a routerLink="/label" routerLinkActive="active" clrVerticalNavLink>Labels</a>
+          <a routerLink="/list" routerLinkActive="active" clrVerticalNavLink>Lists</a>
         </clr-vertical-nav-group-children>
       </clr-vertical-nav-group>
 
-      <div class="nav-divider"></div>
+      <cds-divider cds-layout="m-y:md"></cds-divider>
```
Comment (mathisscott) [inline packages/angular/projects/clarity-adoption/src/app/app.component.html:45]: 🙌

File: packages/angular/projects/clarity-adoption/src/app/pages/accordion/accordion.page.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<h1>Accordion</h1>
+
+<h2>4.0.0</h2>
+
+<h3>Eslint plugin</h3>
+<p>
+  There is <b>eslint plugin</b> that could help with the adoption of Core Alerts. By locating and pinpoint the places
```
Comment (mathisscott) [inline packages/angular/projects/clarity-adoption/src/app/pages/accordion/accordion.page.html]: `There is an...`

`...the adoption of Core Accordions.`

`By locating and pinpointing...`

File: packages/angular/projects/clarity-adoption/src/app/pages/checkbox/checkbox.page.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<h1>Checkbox</h1>
+
+<h2>4.0.0</h2>
+
+<h3>Eslint plugin</h3>
+<p>
+  There is <b>eslint plugin</b> that could help with the adoption of Core Alerts. By locating and pinpoint the places
```
Comment (mathisscott) [inline packages/angular/projects/clarity-adoption/src/app/pages/checkbox/checkbox.page.html]: `There is an...`

`...with the adoption of Core Checkboxes.`

`By locating and pinpointing...`

File: packages/angular/projects/clarity-adoption/src/app/pages/checkbox/checkbox.page.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+@Component({
+  selector: 'app-checkbox',
+  templateUrl: './checkbox.page.html',
+})
+export class CheckboxPage {
+  indeterminate = false;
+
+  code = [
+    // {
```
Comment (mathisscott) [inline packages/angular/projects/clarity-adoption/src/app/pages/checkbox/checkbox.page.ts]: Big commented out stuff still needed?

**Review by mathisscott — CHANGES_REQUESTED** [review]

**Review by mathisscott — APPROVED** [review]

**Review by sis0k0 — APPROVED** [review]

**Pull request merged**